.github/workflows/gradle.yml

```diff
@@ -0,0 +1,47 @@
+name: "Publish to Gradle Gallery"
```

Comment:
I suggest adding to thsi workflow a description, explaining why we created this workflow as a cron, and explain the entire release flow.